src/plugins/intel_cpu/src/nodes/rdft.cpp

```diff
@@ -0,0 +1,1537 @@
+                if (i < input_size) {
+                    dst[2 * i] = src[2 * i];
+                    dst[2 * i + 1] = src[2 * i + 1];
+                } else {
+                    size_t src_idx = 2 * input_size - 2 - i;
+                    dst[2 * i] = src[2 * src_idx];
+                    dst[2 * i + 1] = -src[2 * src_idx + 1];
+                }
+        });
+    }
+}
+
+static void fft_copy_real_input_data(float* dst, float* src, size_t input_size, bool parallelize) {
+    if (!parallelize) {
+        for (size_t i = 0; i < input_size; i++) {
+            dst[2 * i] = src[i];
+            dst[2 * i + 1] = 0;
+        }
+    } else {
+        parallel_for(input_size, [&] (size_t i) {
+            dst[2 * i] = src[i];
+            dst[2 * i + 1] = 0;
+        });
+    }
+}
+
+static void fft_copy_inverse_real_output(float* dst, float* src, size_t num_samples, bool parallelize) {
+    if (!parallelize) {
+        for (size_t i = 0; i < num_samples; i++) {
+            dst[i] = src[2 * i];
+        }
+    } else {
+        parallel_for(num_samples, [&] (size_t i) {
+            dst[i] = src[2 * i];
+        });
+    }
+}
+
+static void fft_copy_inverse_output(float* dst, float* src, size_t num_samples, bool parallelize) {
+    if (!parallelize) {
+        for (size_t i = 0; i < num_samples; i++) {
+            dst[2* i] = src[2 * i];
+            dst[2* i + 1] = src[2 * i + 1];
+        }
+    } else {
+        parallel_for(num_samples, [&] (size_t i) {
+            dst[2* i] = src[2 * i];
+            dst[2* i + 1] = src[2 * i + 1];
+        });
+    }
+}
+
+struct RDFTJitExecutor : public RDFTExecutor {
+    RDFTJitExecutor(bool inverse) : is_inverse(inverse) {
+        enum dft_type rdft_type = is_inverse ? complex_to_real : real_to_complex;
+        if (mayiuse(cpu::x64::avx512_core)) {
+            rdft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::avx512_core>(is_inverse, rdft_type));
+            dft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::avx512_core>(is_inverse, complex_to_complex));
+            fft_kernel.reset(new jit_fft_kernel_f32<cpu::x64::avx512_core>(is_inverse));
+            vlen = cpu_isa_traits<cpu::x64::avx512_core>::vlen;
+        } else if (mayiuse(cpu::x64::avx2)) {
+            rdft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::avx2>(is_inverse, rdft_type));
+            dft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::avx2>(is_inverse, complex_to_complex));
+            fft_kernel.reset(new jit_fft_kernel_f32<cpu::x64::avx2>(is_inverse));
+            vlen = cpu_isa_traits<cpu::x64::avx2>::vlen;
+        } else if (mayiuse(cpu::x64::sse41)) {
+            rdft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::sse41>(is_inverse, rdft_type));
+            dft_kernel.reset(new jit_dft_kernel_f32<cpu::x64::sse41>(is_inverse, complex_to_complex));
+            vlen = cpu_isa_traits<cpu::x64::sse41>::vlen;
+        } else {
+            IE_THROW() << "Can't create RDFT kernel";
```

Comment:
There can be two solutions:
1. Implement some reference here in the CPU plug-in,
2. Add the same check in the function _isSupportedOperation_ and fallback on core reference.